x/datapool/keeper/pool_test.go

```diff
@@ -230,17 +230,18 @@ func (suite *poolTestSuite) TestSetDataCertificate() {
 }
 
 func (suite *poolTestSuite) TestSellData() {
-	poolID := uint64(1)
-	round := uint64(1)
-	dataHash := []byte("dataHash")
-
 	validatorAccount := suite.AccountKeeper.NewAccountWithAddress(suite.Ctx, dataVal1)
 	err := validatorAccount.SetPubKey(dataValPubKey)
 	suite.Require().NoError(err)
 	suite.AccountKeeper.SetAccount(suite.Ctx, validatorAccount)
 
-	pool := makeTestDataPool(poolID)
-	suite.DataPoolKeeper.SetPool(suite.Ctx, pool)
+	suite.TestCreatePool()
+
+	pool, err := suite.DataPoolKeeper.GetPool(suite.Ctx, 1)
+	suite.Require().NoError(err)
+	poolID := pool.PoolId
+	round := pool.Round
+	dataHash := []byte("dataHash")
```

Comment:
It called `CreatePool` because it need to initialize the supply for token minting.